src/color/tier.json

```diff
@@ -73,6 +73,20 @@
               "wcag": "n/a",
               "deprecated": false
             }
+          },
+          "lounge": {
+            "value": "{color.brand.midnight.400.value}",
```

Comment:
midnight is a deprecated tier 1 token name. Please work with @leeejune to make sure we are pointing this to the correct "new" tier 1 token name(s).

Reply:
Hi @jason-capsule42, to overcome that deprecated token name I added another color brand token named `lounge` with the value of `01426a`. Please let me know if there’s another concern.
CC. @leeejune